### PR TITLE
Fixed openQA issue

### PR DESCRIPTION
**Pull request by mchf**
bnc#930448

### CODE REVIEW

File: src/modules/LanItems.rb [hunk 1]
```diff
@@ -1229,7 +1229,7 @@ def BuildBondIndex
       deep_copy(index)
     end
 
-    def startmode_overview
+    def startmode_overview(item_id)
```
Comment (mvidner) [inline src/modules/LanItems.rb]: \# @param item_id [Integer] a key for {#Items}

I keep forgetting which API uses numbers and which uses the kernel names.

Comment (mvidner) [inline src/modules/LanItems.rb]: NTH: Consider adding this to the many other methods which are missing an explanation for item_id / itemId.

File: src/modules/LanItems.rb [hunk 2]
```diff
@@ -1257,7 +1257,8 @@ def startmode_overview
         )
       }
 
-      startmode_descr = startmode_descrs[NetworkInterfaces.Current["STARTMODE"].to_s] || _("Started manually")
+      ifcfg = GetDeviceMap(item_id) || []
```
Comment (mvidner) [inline src/modules/LanItems.rb]:
```
$ ruby -e '[]["foo"]'
-e:1:in `[]': no implicit conversion of String into Integer (TypeError)
        from -e:1:in `<main>'
```

**Comment by mvidner** [comment]
LGTM

**Comment by mchf** [comment]
Thanks for reviews.

**Pull request merged**